web/app/adapters/application.ts

```diff
@@ -0,0 +1,22 @@
+import JSONAdapter from "@ember-data/adapter/json-api";
+import { inject as service } from "@ember/service";
+import ConfigService from "hermes/services/config";
+import FetchService from "hermes/services/fetch";
+import SessionService from "hermes/services/session";
+
+export default class ApplicationAdapter extends JSONAdapter {
+  @service("config") declare configSvc: ConfigService;
+  @service("fetch") declare fetchSvc: FetchService;
+  @service declare session: SessionService;
+
+  get namespace() {
+    return `api/${this.configSvc.config.api_version}`;
+  }
+
+  get headers() {
+    return {
+      "Hermes-Google-Access-Token":
+        this.session.data.authenticated.access_token,
+    };
+  }
```

Comment:
Are these needed/used?

Reply:
These will come into play when we start fetching with `this.store` methods.